change/@fluentui-react-tabs-c6386b10-6c09-4856-84a4-c5519c849fc3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31775&commitId=f221b499bb26433e3f40546464af1a35201df187&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+inactive+badge.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.appearance - High Contrast.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Image Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Shape.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>